sirepo/http_reply.py

```diff
@@ -283,6 +283,8 @@ def _as_attachment(resp, content_type, filename):
 
 def _gen_exception_error(exc):
     pkdlog('unsupported exception={} msg={}', type(exc), exc)
+    if sirepo.http_request.is_spider():
+        raise werkzeug.exceptions.InternalServerError
```

Comment:
I think this needs to be `raise werkzeug.exceptions.InternalServerError()`.

Reply:
I was curious. Looks like it can be either https://docs.python.org/3.7/tutorial/errors.html#raising-exceptions

Reply:
85a40e1

We use ```raise Whatever()``` everywhere else so I changed it

Reply:
Consistency is better. Looks unfinished otherwise. Semantics would be different btw with:

```py
def f():
   return ValueError
```

and

```py
def f():
    return ValueError
```

In the first case, the exception is probably created inside the call to `f`. In the second, the exception would be created at the time `raise` was called with the return value of `f`.

Python is very strange...